src/pages/workspace/members/WorkspaceInviteMessageComponent.tsx

```diff
@@ -230,7 +232,22 @@ function WorkspaceInviteMessageComponent({
                     </View>
                     <View style={[styles.mb3]}>
                         <View style={[styles.mhn5, styles.mb3]}>
-                            {shouldShowMemberNames && (
+                            {isInviteNewMemberStep && (() => {
+                                const invitingMemberEmail = Object.keys(invitedEmailsToAccountIDsDraft ?? {}).at(0) ?? '';
+                                const memberDetails = getPersonalDetailByEmail(invitingMemberEmail);
+                                const memberName = Str.removeSMSDomain(memberDetails?.displayName ?? '');
+                                return (
```

Comment:
We shouldn't really have these nested `return` statements like this. I think you should move the three variables to be outside of `return()` and then remove the `return()` here.